specs/pipelines/cluster-resolvers.spec

```diff
@@ -14,19 +24,7 @@ Type: Functional
 Importance: High
 
 Steps:
-    * Create project "releasetest-tasks"
-    * Verify ServiceAccount "pipeline" exist
-    * Create
-      |S.NO|resource_dir                                                |
-      |----|------------------------------------------------------------|
-      |1   |testdata/resolvers/tasks/resolver-task2.yaml                |
-    * Create project "releasetest-pipelines"
-    * Verify ServiceAccount "pipeline" exist
-    * Create
-      |S.NO|resource_dir                                                |
-      |----|------------------------------------------------------------|
-      |1   |testdata/resolvers/pipelines/resolver-pipeline.yaml         |
-    * Create project "releasetest-pipelineruns"
+    * Switch to autogenerated namespace
```

Comment:
Do you really need this step? Won't it use autogenerated namespace automatically?